Feature/gm 7669

Tai-DucTran opened the pull request:
Support more customized styles from floating placeholder
Support self-close the popover

farhanabi commented on test/e2e/passwordInput/passwordInput.spec.ts-snapshots/passwordinput-chromium-linux.png:
This remove the placeholder?

Tai-DucTran replied:
Found the issue @farhanabi 
I removed `{...props}` in <Input>

farhanabi commented on test/e2e/passwordInput/passwordInput.spec.ts-snapshots/passwordinput-disabled-chromium-linux.png:
Ditto. This remove the placeholder?

Tai-DucTran replied:
Found the issue @farhanabi 
I removed `{...props}` in <Input>

farhanabi replied:
Noice, so it won't change the existing behaviour?

Tai-DucTran committed:
Input: added more custom props to customize floating style
floating placeholder color
floating placeholder top position
floating placeholder error color

Tai-DucTran committed:
TextArea: add new props to support customize floating placeholder styles

Tai-DucTran committed:
Select: ensure component stay consitent when having selected value

Tai-DucTran committed:
Select: allow external component close the popover
